main.py

```diff
@@ -124,6 +133,9 @@ async def send_session_update(openai_ws):
             "turn_detection": {"type": "server_vad"},
             "input_audio_format": "g711_ulaw",
             "output_audio_format": "g711_ulaw",
+            "input_audio_transcription": {
```

Comment:
Does this affect the behavior of the model in any way?

Reply:
Doesn't seem like it https://platform.openai.com/docs/guides/realtime/input-and-output-transcription